SECURITY.md

```diff
@@ -1,49 +1,25 @@
-<!--- https://www.eclipse.org/security/ --->
-_ISO 27005 defines vulnerability as:
- "A weakness of an asset or group of assets that can be exploited by one or more threats."_
+<!--
+    For any questions about implementing security best practices, contact the
+    Eclipse Foundation Security Team at security@eclipse-foundation.org
+-->
 
-## The Eclipse Security Team
+# How To Report a Vulnerability
 
-The Eclipse Security Team provides help and advice to Eclipse projects
-on vulnerability issues and is the first point of contact
-for handling security vulnerabilities.
-Members of the Security Team are committers on Eclipse Projects
-and members of the Eclipse Architecture Council.
+If you think you have found a vulnerability in Eclipse EE4J you can report it using one of the following ways:
 
-Contact the [Eclipse Security Team](mailto:security@eclipse.org).
+* Contact the [Eclipse Foundation Security Team](mailto:security@eclipse-foundation.org)
+* Create a [confidential issue](https://gitlab.eclipse.org/security/vulnerability-reports/-/issues/new?issuable_template=new_vulnerability)
 
-**Note that, as a matter of policy, the security team does not open attachments.**
+You can find more information about reporting and disclosure at the [Eclipse Foundation Security page](https://www.eclipse.org/security/).
 
-## Reporting a Security Vulnerability
+# Supported Versions
 
-Vulnerabilities can be reported either via email to the Eclipse Security Team
-or directly with a project via the Eclipse Foundation's Bugzilla instance.
+<!--
+    Which releases of the project's software are actively maintaned and receive security updates?
+-->
+Supported versions are:
+* Jakarta EE 10
```

Comment:
This is not correct. EE4J is a GH org mostly for implementations, some of which are not tied directly to any Jakarta EE release. The Jakarta EE related repositories have moved, or is in the process of moving to the Jakarta EE GH org.

Reply:
Pardon my ignorance, but what should be in there instead?
@mrybczyn suggested we can also not include this section on this repo but have a separate SECURITY.md for each project repo

Reply:
Maybe something like: "Check the individual repositories for versions supported."